tests/endpoints/echo/server/app.go

```diff
@@ -350,7 +350,7 @@ func authInfoHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	b, err := base64.StdEncoding.WithPadding(base64.StdPadding).DecodeString(encodedInfo)
```

Comment:
Isn't this a breaking change? Can you explain why this is changed here?

Reply:
Jason reverted padding change in upstream envoy.  this envoy got that change.  So we just revert the behaviors back.  Jason still works on the pr to add a flag to add padding in upstream envoy.  Once that change is in, we can add flag to enable it.

Reply:
Got it. Well I guess reverting the breaking change is OK